code/_helpers/unsorted.dm

```diff
@@ -311,26 +311,20 @@ Turf and target are seperate in case you want to teleport some distance from a t
 					break
 
 		//update our pda and id if we have them on our person
-		var/list/searching = GetAllContents(searchDepth = 3)
-		var/search_id = 1
-		var/search_pda = 1
+		var/list/searching = src.GetAllContents()
```

Comment:
no src.

Why did you get rid of the searchDepth option in GetAllContents? It's deliberately there to prevent way too much recursion from happening